Sources/Plasma/CoreLib/hsRefCnt.h

```diff
@@ -171,7 +171,7 @@ class hsWeakRef
 
     hsWeakRef<_Ref> &operator=(_Ref *obj)
     {
-        fObj = obj.Get();
+        fObj = obj;
```

Comment:
This one has always been an error on Mac/Clang/Xcode. I wasn't sure if there was some trick making it work for everyone else but me. On master, this has been fixed, so I'm just grabbing the fix and committing it separately.